subjects/argsort/README.md

```diff
@@ -1,30 +1,27 @@
 ## argsort
 
-### Instruction
+### Instructions
 
 Write a program where you define if an argument is sorted or not sorted:
-- Your Program should return T if an argument is sorted.
-- Your Program should return F if an argument is not sorted.
+- Your Program should return `T` if an argument is sorted.
+- Your Program should return `F` if an argument is not sorted.
 - If The argument is empty nothing will be printed.
-- If the number of arguments is not `1` Print nothing.
-- If the length argument is equal to `1` Print nothing.
-- Your program should always print `('\n') `at the end of the output.
+- If the number of arguments is not 1 Print (`'\n'`).
+- If the length argument is equal to 1 Print(`'\n'`).
+- Your program should always print (`'\n'`) at the end of the output.
 
 ### Usage
 
 And it's output should be:
 
 ```console
-$ go run . | cat -e
-$
+$ go run . 
 $ go run . a | cat -e
-$
 $ go run . " 5ABc" | cat -e
 T$
 $ go run . "zA1" | cat -e
 F$
 $ go run . "01Talent" "student" | cat -e
-$
 $ go run . "Hello World" | cat -e
 F$
 $ go run .  "a b c" | cat -e
```

Comment:
you need to explain in the readme, why this is not considered a sorted arg

Reply:
done

Reply:
Where is it explained?
Also, what happens if you pass just spaces as args?
This is not consistent, when you have spaces in front it returns F but when its behind it says it is sorted...
If you trim the spaces in front, maybe you should trim them behind as well, or else you need to write the rules in the readme
```
$ go run .  "    "
T
$ go run .  "    s      "
F
$ go run .  "    s"
T
$ go run .  "s    "
F
```

Reply:
You need to explain that on the readme, otherwise we will be flooded with questions from the students and we need to avoid that. 
Just say that all printable characters will be used